packages/editor-ui/src/components/Node.vue

```diff
@@ -106,9 +113,31 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 
 			return workflowResultDataNode.length;
 		},
+		nodeOffsetPosition() {
+			return this.$store.getters.getNodeViewOffsetPosition;
+		},
 		isExecuting (): boolean {
 			return this.$store.getters.executingNode === this.data.name;
 		},
+		isDragActiveTriggerNode (): boolean {
```

Comment:
would rename to `isDragActive` since this is tracking any drag across all nodes in workflow

Reply:
Yup, sounds more logic. Renamed.